### PR TITLE
Trunk 4986:Upgrade liquibase to 3.5.x version

### PR DESCRIPTION
**Pull request by jwnasambu**
## Description of what I changed
I upgraded liquibase to 3.5.x version 

## Issue I worked on
https://issues.openmrs.org/browse/TRUNK-4986

### CODE REVIEW

**Comment by jwnasambu** [comment]
@dkayiwa  and @ssmusoke I have tried to upgrade liquidate to 3.5.x version as per the ticket description  but am getting the same error log  when I upgrade it from version 3.0.0 - 3.8.4 as reflected on this paste bin link https://pastebin.com/ex9tuKdA. even after ensuring the liquidate extensions are up to date. What could be the problem?

**Comment by gitcliff** [comment]
helo @jwamalwa kindly dont forget to squash the commits

**Comment by jwnasambu** [comment]
@gitcliff  thanks for the correction. I had not noticed it and besides they are different issues how I mixed the two I don't know.

File: api/src/main/java/org/openmrs/validator/PersonAddressValidator.java [hunk 1]
```diff
@@ -62,13 +62,10 @@ public void validate(Object object, Errors errors) {
...
-		}
-		
 		PersonAddress personAddress = (PersonAddress) object;
-		
+		if (personAddress == null) {
+           errors.rejectValue("value", "error.null");
+		}
 		//resolve a shorter name to display along with the error message
 		String addressString;
 		if (StringUtils.isNotBlank(personAddress.getAddress1())) {
```
Comment (saadkhaleeq610) [inline api/src/main/java/org/openmrs/validator/PersonAddressValidator.java]: Rename function name getAddress1 to more self-explanatory name

Comment (f4ww4z) [reply api/src/main/java/org/openmrs/validator/PersonAddressValidator.java]: @saadkhaleeq610 Did you see the PR title? Renaming functions is not part of it.

**Pull request closed without merging**